src/models/blockchain/block/mod.rs

```diff
@@ -557,20 +575,16 @@ mod block_tests {
 
     use crate::{
         config_models::network::Network,
+        database::storage::storage_schema::SimpleRustyStorage,
+        database::NeptuneLevelDb,
         models::{
             blockchain::transaction::PublicAnnouncement, state::wallet::WalletSecret,
             state::UtxoReceiverData,
         },
         tests::shared::{get_mock_global_state, make_mock_block, make_mock_block_with_valid_pow},
+        util_types::mutator_set::archival_mmr::ArchivalMmr,
     };
-    use tasm_lib::twenty_first::{
-        storage::level_db::DB,
-        util_types::{
-            emojihash_trait::Emojihash,
-            mmr::archival_mmr::ArchivalMmr,
-            storage_schema::{SimpleRustyStorage, StorageWriter},
-        },
-    };
+    use tasm_lib::twenty_first::util_types::emojihash_trait::Emojihash;
```

Comment:
Note that trait Emojihash has been killed in twenty-first v39.

Reply:
yeah, so that will affect RPC and dashboard display, at least.  I'm not sure what else.    seems outside the scope of this PR.

I'm thinking its best to merge this PR as-is, which depends on a specific twenty-first github revision prior to emojihash removal.

Then we can remove it from neptune-core in a much smaller followup commit/pr.